src/utils/resolveL1OpStackContractAddress.test.ts

```diff
@@ -0,0 +1,46 @@
+import type { Chain } from 'viem'
+import { expect, test } from 'vitest'
+import { publicClient } from '../_test/utils.js'
+import { base } from '../chains/base.js'
+import { OpStackL1Contract } from '../types/opStackContracts.js'
+import { resolveL1OpStackContractAddress } from './resolveL1OpStackContractAddress.js'
+
+test('resolves Base OptimismPortal contract address', async () => {
```

Comment:
we should also check that it just returns `address` if `address` is passed. 

Thanks for this! We had coverage here from the tests on writeUnsafeDepositTranasction, but good to have these. 

https://github.com/base-org/op-viem/blob/main/src/actions/wallet/L1/writeUnsafeDepositTransaction.test.ts#L170-L232

Reply:
Good call.